cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -95,12 +95,18 @@ object ReporterCommand : CommandWithHelp() {
         resolutionsFile?.readValue<Resolutions>()?.let { resolutionProvider.add(it) }
 
         reportFormats.distinct().forEach {
+            val name = it.toString().substringBefore("Reporter")
```

Comment:
I'd prefer to use `removeSuffix()` like we do at https://github.com/heremaps/oss-review-toolkit/blob/master/cli/src/main/kotlin/commands/ReporterCommand.kt#L50.

Reply:
Done.

Reply:
Done.

Reply:
Done.

Reply:
Done.